core/src/sigverify_stage.rs

```diff
@@ -216,27 +202,11 @@ impl SigVerifierStats {
     }
 }
 
-impl SigVerifier for DisabledSigVerifier {
```

Comment:
Maybe add a note into the commit description as to why you are removing this trait?
Looks like it had only one use case - part of the API of `TransactionSigVerifier`.
Was it an unnecessary abstraction?

Reply:
> Was it an unnecessary abstraction?

Yes, made things a little easier with this change. I intentionally have that change broken out into its' own commit, and should we proceed with this PR, I will open a separate PR for just that commit and rebase this PR on top